### PR TITLE
7.2 Deployment | Solr | LRDOCS-6986 | LRDOCS-7151

### PR DESCRIPTION
**Pull request by rbohl**
SOLR RELEASE DEPENDENCY: https://issues.liferay.com/browse/LRDOCS-6986 and LRDOCS-7151 (let's do the review work only on the linked ticket). 

_Edited to add:_
The target date for the Solr release is August 6th, to release it with FP-1, which it's now dependent on.

### CODE REVIEW

**Comment by BryanEngler** [comment]
hey @rbohl, i just talked to tibor and there were a few more things added to the "unsupported" list on https://issues.liferay.com/browse/LPS-96522

File: deployment/articles/03-installing-a-search-engine/03-solr/03-installing-solr-solrcloud.markdown [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+   cluster. Use this command:
+
+        ./bin/solr -e cloud
+
+5. Complete the setup wizard. These steps demonstrate creating a two-node
+   cluster:
+
+    -  Enter `2` for the number of nodes.
+    -  Specify ports `8983` and `7574` (the defaults). Both nodes are
+       started with the start commands printed in the log:
```
Comment (lipusz) [inline deployment/articles/03-installing-a-search-engine/03-solr/03-installing-solr-solrcloud.markdown]: @rbohl 

We are missing the command for the 2nd node:

```bash
Starting up Solr on port 8983 using command:
"bin/solr" start -cloud -p 8983 -s "example/cloud/node1/solr"

Starting up Solr on port 7574 using command:
"bin/solr" start -cloud -p 7574 -s "example/cloud/node2/solr" -z localhost:9983
```

Comment (rbohl) [reply deployment/articles/03-installing-a-search-engine/03-solr/03-installing-solr-solrcloud.markdown]: It was intentional. I just saw no point in pasting in both of them.

**Comment by lipusz** [comment]
Let's add it I think.

**Comment by lipusz** [comment]
Just in case someone is using our docs and not Solr's to see how to start cloud

File: deployment/articles/03-installing-a-search-engine/01-intro.markdown [hunk 2]
```diff
@@ -21,39 +21,76 @@ Learn to configure a remote Elasticsearch server or cluster
...
-Otherwise, feel free to use Elasticsearch or Solr to index your portal content.
+Elasticsearch and Solr are both supported, but there are limitations to
+Liferay's Solr integration. To make use of some features, you must choose
+Elasticsearch. 
+
+### End User Feature Limitations of Liferay's Solr Integration
+
+- [Liferay Commerce](https://help.liferay.com/hc/en-us/articles/360017869952)
+- [Workflow Metrics](/docs/7-2/user/-/knowledge_base/u/workflow-metrics-the-service-level-agreement) 
+- [Custom Filter search widget](/docs/7-2/user/-/knowledge_base/u/filtering-search-results-with-the-custom-filter-widget)
```
Comment (rbohl) [inline deployment/articles/03-installing-a-search-engine/01-intro.markdown]: @lipusz do we need to add the federated search/Low Level Search Options widget here?

**Comment by sez11a** [comment]
Just started reviewing :)

**Pull request merged**